plenum/server/consensus/ordering_service.py

```diff
@@ -211,7 +211,13 @@ def __init__(self,
         self._stasher.subscribe(PrePrepare, self.process_preprepare)
         self._stasher.subscribe(Prepare, self.process_prepare)
         self._stasher.subscribe(Commit, self.process_commit)
+        self._stasher.subscribe(ApplyNewView, self.process_apply_new_view)
```

Comment:
No, it's going to be stashed. Please note that `process_apply_new_view` is not fully implemented yet (will be in next PR). It will use `OrderingServiceMsgValidator#validate_new_view` which may stash future view messages.